### ISSUE TITLE
Accuracy is not shown in the logs.

### ISSUE THREAD

**Issue by alibi123** [google-research/text-to-text-transfer-transformer#481]
I've added multiple metrics from `t5.evaluation.metrics` but I cannot see them in the logs. All I can see in the logs is:
```
step, tf_loss[0.00107303483][1000182]
step, tf_loss[0.001391956][1000183]
step, tf_loss[0.00186035747][1000184]
step, tf_loss[0.00131111126][1000185]
step, tf_loss[0.00150477933][1000186]
```

And here's how I've added data task:
```
t5.data.TaskRegistry.add(
        "qa_gen",
        t5.data.TextLineTask,
        split_to_filepattern=OUT_SPLIT,
        text_preprocessor=[
            functools.partial(
                t5.data.preprocessors.parse_tsv,
                field_names=["answer", "question"]),
            qa_gen_preprocessor,
        ],
        postprocess_fn=t5.data.postprocessors.lower_text, 
        metric_fns=[t5.evaluation.metrics.accuracy, t5.evaluation.metrics.bleu, t5.evaluation.metrics.rouge],
        output_features=output_features,
    )
```

```
model = t5.models.MtfModel(
        model_dir=MODEL_DIR,
        tpu=None,
        mesh_shape='model:1,batch:1',
        mesh_devices=['gpu:0'],
        batch_size=4,
        model_parallelism=1,
        sequence_length={"inputs": 256, "targets": 32},
        learning_rate_schedule=0.003,
        save_checkpoints_steps=5000,
        iterations_per_loop=100,
    )

    model.finetune(
        mixture_or_task_name="qa_gen",
        pretrained_model_dir=PRETRAINED_DIR,
        finetune_steps=FINETUNE_STEPS
    )
```

Why I can't see accuracy metrics?

**Comment by craffel** [comment]
You need to eval the model. It will not print out accuracy or other metrics unless it is in eval mode.
https://github.com/google-research/text-to-text-transfer-transformer/blob/master/t5/models/mtf_model.py#L265
